drivers/watchdog/wdt_cmsdk_apb.c

```diff
@@ -65,8 +65,9 @@ struct wdog_cmsdk_apb {
 /* Keep reference of the device to pass it to the callback */
 const struct device *wdog_r;
 
-/* watchdog reload value in sec */
-static unsigned int reload_s = CMSDK_APB_WDOG_RELOAD;
+/* watchdog reload value in clock cycles */
+static unsigned int reload_cycles = CMSDK_APB_WDOG_RELOAD;
+static uint8_t assigned_channels;
```

Comment:
It seems there could only be 1 channel. Wouldn't it make more sense to use a boolean here instead?